examples/wind_example.ipynb

```diff
@@ -0,0 +1,925 @@
DDaWUEkJufvnll7MSEhLi5s2bN2gjV7bsKu3l5YXU1FScOnUKYWFhGDVqlFXPP9TV1bfCYuZhMJpRdL4BQPsQ946dp1BR3YjVt0xHQEdNkLff2IOcM5VwchIharQ/Fl0/+Au7H0uYj8cS5nd9zfE8Np0+jnqtBo9NnA4/l99fH3KNFg9t/x5GC4cfcguR+dSD3S7s7c7Xh0+jUd3eiuarQ1l45dZrrPtE+ojjeKx/ejuMRguOHz+P+Lhg+A/xWi39JZfLce7cOaSkpFi9o7Q9p5OKi4uxevXqeoVCcRVLXqzDRr2QHAqbQuolSqlBqVQuuPXWW+VlZWWDck1bJi+dpFIppk2bhqamJpw9exaO3Auqr2JjAjFp4ii4ukjwwN2zAQDHTpTg863p2PtrHv7vjd9bfMg8XSAUta87cR8iO1q+KynA+6czsL3oLJ46svei+3j+93c3C8+jNyOt2SU1WPTiZpwtq4NELIJELEJa7JUL5w0GQgiEXVMoBALB8NtBRylFSUkJSkpKkJaWZvXkpZM9ppNUKhUWLlzY1NDQsIhSqhyUizIjAhuB6QNKaT0hZPE111yzNysry6e7QlLWMhjJSyehUIiUlBSUlJQgIyMDkyZNgpOTk02vORQIhQK88sINF32va60BwUV/KP+y/lqMiQ2Cr687Zs6KHcwwL/F1/hn84+RRRHRszyYgl4yuBMnc8eaya7C34DxunzLhggTg8v7+34OobVLDWSzCPVdNwvwJMYgOsv/uO4GAYMM7q/DTz2cxLS0Gvr62+72zB47jcObMGYhEIqSlpdl8d+BgjsRwHIfrr7++RS6Xr6WU2r5j6EjCln+wBKavKKWnZDLZ4zfccMPGffv2edrizWYwk5dOnYt73d3dcezYsV73UBpuUidH4c/3zEFlVSNuvfH3Am8uLhLcdEvvKtba2svHDsHAWaA3m3Hj2HEQEoKHU9IuOe7qsWNw9dgxvT7v2DB/1DS2glIgNS5iSCQvnWKiA/HIw4H2DsPq2trakJ2djdDQUERGRg7adQcriXnsscc0hYWFn2m12u02ucBIRQEycgbLL4slMP2gUqm2+Pv7T1y3bt1d7733nlU/DtojeblQYGAg3N3dkZ2djeDgYERFRY2ooneEECxdNLQKF/5RnI8fipsbISAEDyenXbT2ZSCev+UqzBkfjSAvd8SFsd5ZtlZXV4fi4uKLGjIOJlsnMR9++KFh+/btGUql8mmrnphhOrAEpp+USuWj//3vf8NDQkL+9Mwzz7hY45z2Tl46ubq6Ytq0aSgoKEBmZiaSk5NHxJSSo/jP9TfiWF0V4n38rJa8AIBIKMDc8dZpNUEpRWVdM3w8XeHueuWNe1WNrdiTdQ5TY8KREhVilesPZRzHIS8vD0ajEdOnT+9VZV1bsVUS891331n+9re/nVMqlazSrq2wKSSWwPRXx86kGzds2HAwKCho0urVqwdUfGOoJC+dhEIhxo0bh/r6ehw9enTEbrW2BXWbAY+//z0UrVo8fuMs5FfJMT4yCDPHRXUdQym97MiXVCzG/IieE41WvQG/FJxHYnBAn7ZRW8Pb/zqAHw/lQSwSYuubdyDAp/tFqZRS3LFpO5rbdPj84Cn88PRqBHoOrzUuF9JoNDh9+nTXrr+hMLpp7STm8OHD/Jo1a8qVSuU8SqnBGjEyTHdYAjMAlFIzIeTq9evXn/D394+79tpr+1UcZKglLxcKCgqCTCZDdnY2/P39ERMTMyTedB3ZnoxzyCtvgNnC4ZnNP8HM8RCLhNjy1C2ICfHFDycL8OK2XxHi7YEtj94MT9f+7Xpa/eU3KG9qBgDcNW0ialrUWDs7FZE+tl/bdDjzPIwmCwghyDtff9kEBgDajKaOOloUBvPQLig4ENXV1SgtLUVycvJF/bGGAmslMbm5ubjpppvqlErlHEppqxVDZP6IDcCwbdQDRSltUyqVs++6666qEydO9PnxQzl56eTi4oJp06bBYrEgIyMDer3e3iE5tNHBPhAQAqmTGAKBADylIATQGdvLY3y0NwMWjodcpcVv+eX9vk6dSg2jhQOlwOfHs/BjXiHWbv++5wf+gc5kxoZ96Xj/f8dhsvQuwbhz6VQICIGflyumjLv8VmxCCDbdvQSpMeF4YvFsjPKzfXJVVtOIJ97aiU92HL1ou7mtmM1mZGdnQ6FQYPr06UMueek00C3WFRUVWLhwoVwul8+nlNbZIETmAoRSq98cDRuBsQJKaRMhZM7SpUszDh48GBQX17tKoY6QvHQSCAQYO3YslEolTpw4gcjISISHh7PRmH6YEh+Oz9avRKNKBy8PKT796QQmjwlFUlR7c8TZiVH49lguBIRg3Kj+D+e/dcNCvPXrb4jy9cLh0goQQiDuYzNAAPjnL+nYnnm2qx7LmrlTL3vsvvJiPHboJwS7eWDXZ/fDz8Wtx/OnjglH6pjwPsfVX8+9+yPKa5qQVVCN+NGBmDnROut+uqNQKJCfn4+YmBiEhIQM+d+X/o7ENDY2Yv78+Y21tbXXU0qLbRkjw3RiCYyVUEqrCCF/+tOf/nTg+PHjfiEhV16M6EjJy4X8/Pwwffp0FBQUoL6+HklJSZBKh0Zht8HA8Txe+ewXZJ+rxsM3z8KCqf2rCTN21O9bgt9du/Si+55cOhtLpybAx90VPu79Xx8+OyYSs2Pat+buzilAXr0Cd6Wl9Pk8/AWfzLgeRiz+cSodOosZNRoVfq0sxar4pD5fDwA+LzmCirZGPBG/EB5O1n19ublIIBQQUErh5mKbvlFmsxl5eXkwmUxIS0vrtkHnUNXXJEar1WLu3LnNtbW1t1NKTw5GjAzYIl6wBMaqKKV5IpHohrlz5+46ceKEz+XqqDhq8tJJLBYjKSkJSqUSGRkZGD16NMLCwob8p8v+snA83vzkV5wrbcD1C8bh4Mnz0BvN+PsX+/udwFwJIQRjgq3bf2hJ0lgsSRoLAMhTytGs12NGWES37QUUei3azCZEerRv7X3
```

Comment:
I don't really think `daily_data` is a good name here. Shouldn't it be `Jan_data`?

Reply:
Agreed, I renamed the daily/monthly variables to ``Jan_data`` and ``Jan_11_data`` for clarity